blockbuster/abci/abci.go

```diff
@@ -132,8 +132,6 @@ func ChainPrepareLanes(chain ...blockbuster.Lane) blockbuster.PrepareLanesHandle
 				// Write the cache to the context since we know that the lane successfully prepared
 				// the partial proposal.
 				write()
-
```

Comment:
Removing this log since we already have one at the beginning of the chain decorator.